app_dart/test/request_handlers/push_gold_status_to_github_test.dart

```diff
@@ -143,12 +144,13 @@ void main() {
         );
       }
 
-      PullRequest newPullRequest(int number, String sha, String baseRef, {bool draft = false}) {
+      PullRequest newPullRequest(int number, String sha, String baseRef, {bool draft = false, DateTime updated}) {
         return PullRequest()
           ..number = 123
           ..head = (PullRequestHead()..sha = 'abc')
           ..base = (PullRequestHead()..ref = baseRef)
-          ..draft = draft;
+          ..draft = draft
+          ..updatedAt = updated ?? DateTime.now();
```

Comment:
Same comment as above UTC/timezone?

Reply:
I changed the check to always convert the DateTimes to UTC for the comparison. Good catch, thanks!